### PR TITLE
Feature/id 305/send email with api rest

### PR DESCRIPTION
**Pull request by VillalobosGarciaGaby**
What did I do?
- I sent an email to hermes.info.app@gmail.com notifying who is the user requesting to be an administrator.
- I integrated the verification code with the email message.
How did I do it?
I did it with the backend all about aesthetics with html, and with the help of nodemailer. I also called it from android studio to pass the email parameters and verification code.
Why did I do that?
To identify which users want to be "Administrators" and get to have control, who to send email to and who not to send email to.

### CODE REVIEW

File: app/src/main/java/com/isc/hermes/EmailVerificationActivity.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_mail_verification);
         validator = Validator.getValidator();
-        validator.obtainVerificationCode();
+        //validator.obtainVerificationCode();
```
Comment (SamuelEscalera) [inline app/src/main/java/com/isc/hermes/EmailVerificationActivity.java]: If you are not using this line, it should be deleted.

**Comment by SamuelEscalera** [comment]
image


- The verification code I put as a user is barely visible. 
-  The email that appears in the UI is not the one I used to sign in.

File: app/src/main/java/com/isc/hermes/model/signup/SignUpTransitionHandler.java [hunk 2]
```diff
@@ -33,6 +34,9 @@ public void transitionBasedOnRole(User user, Context packageContext) {
     public void sendVerificationCode(UserRoles roles, String email) {
```
Comment (DiegoFigueroaSevillano) [inline app/src/main/java/com/isc/hermes/model/signup/SignUpTransitionHandler.java:41]: this method not have javadoc

File: app/src/main/java/com/isc/hermes/database/ApiRequestHandler.java [hunk 3]
```diff
@@ -69,6 +69,26 @@ public void postDataFromApi(String apiUrl, String jsonData) {
...
+     */
+    public void postDataFromApi(String apiUrl) {
+        try {
+            URL url = new URL(apiUrl);
+            HttpURLConnection connection = (HttpURLConnection) url.openConnection();
+            connection.setRequestMethod("POST");
+            connection.setRequestProperty("Content-Type", "application/json");
+            connection.setDoOutput(true);
+            int responseCode = connection.getResponseCode();
+            System.out.println("CODE RESPONSE: " + responseCode);
```
Comment (DiegoFigueroaSevillano) [inline app/src/main/java/com/isc/hermes/database/ApiRequestHandler.java:85]: Delete this system out

File: app/src/main/java/com/isc/hermes/model/signup/SignUpTransitionHandler.java (hunk 2 above)
Comment (VillalobosGarciaGaby) [reply app/src/main/java/com/isc/hermes/model/signup/SignUpTransitionHandler.java:41]: Done.

File: app/src/main/java/com/isc/hermes/EmailVerificationActivity.java (hunk 1 above)
Comment (VillalobosGarciaGaby) [reply app/src/main/java/com/isc/hermes/EmailVerificationActivity.java]: Done

**Review by SamuelEscalera — APPROVED** [review]

**Review by DiegoFigueroaSevillano — APPROVED** [review]

**Pull request merged**